.github/CODEOWNERS

```diff
@@ -14,8 +14,8 @@
 /src/pages/library/DataBrowser* @DataBiosphere/data-explorer-eng
 /src/pages/library/Datasets* @DataBiosphere/data-explorer-eng
 /src/pages/library/SearchAndFilterComponent* @DataBiosphere/data-explorer-eng
-/src/dataset-builder/* @DataBiosphere/data-explorer-eng
-/src/data-catalog/* @DataBiosphere/data-explorer-eng
+/src/dataset-builder/ @DataBiosphere/data-explorer-eng
```

Comment:
When I was reading about the [syntax](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) for this file, I saw that the asterisk is not recursive. I assume this was not what was intended here.

```
# In this example, @doctocat owns any files in the build/logs
# directory at the root of the repository and any of its
# subdirectories.
/build/logs/ @doctocat

# The `docs/*` pattern will match files like
# `docs/getting-started.md` but not further nested files like
# `docs/build-app/troubleshooting.md`.
docs/*  docs@example.com

Reply:
FYI @pshapiro4broad ^